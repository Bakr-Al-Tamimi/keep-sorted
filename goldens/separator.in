Comma separator:
static class Foo {
  // keep-sorted-test start
  "Foo",
  "Bar",
  "Baz"
  // keep-sorted-test end
}

Comma separator with comments:
// keep-sorted-test start sticky_comments=yes
"Foo",
// Sticky comment
"Bar",
"Baz"
// Trailing comments
// keep-sorted-test end
