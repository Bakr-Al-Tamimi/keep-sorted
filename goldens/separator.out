Comma separator:
static class Foo {
  // keep-sorted-test start
  "Bar",
  "Baz",
  "Foo"
  // keep-sorted-test end
}

Comma separator with comments:
// keep-sorted-test start sticky_comments=yes
// Sticky comment
"Bar",
"Baz",
"Foo"
// Trailing comments
// keep-sorted-test end
